Split/HttpSplitChangeFetcher.swift

```diff
@@ -48,7 +48,7 @@ public enum FecthingPolicy {
             }
             semaphore.wait()
             
-            guard let change: SplitChange = try requestResult!.unwrap() else {
+            guard let change: SplitChange = try requestResult!.unwrap(), change.isValid else {
```

Comment:
Can you check that this error is logged but does not cause this to crash?

Reply:
This function is called from RefreshableSplitFetcher cache and it's the error is handled correctly . No crash should happen here.